### PR TITLE
Update simple-lambda-parameters-with-modifiers.md

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Review by jnm2 — APPROVED** [review]
In the field proposal, open questions were added at the end of _all_ the questions rather than being inserted above the answered ones. In a bigger proposal, flipping this around might make it easier to find the most recent questions when scrolling.

**Comment by CyrusNajmabadi** [comment]
honstly, this is small, and going to ldm next week.  i just want hte questions in there :)

**Comment by alrz** [comment]
> A lambda parameter list still cannot mix `implicit_anonymous_function_parameter` and `explicit_anonymous_function_parameter` parameters.

This could be actually useful where some types can be inferred and some not, it's probably a different scenario than this proposal is trying to address (maybe more related to "partial type inference") but I think it can be folded into the proposed changes here?

File: proposals/simple-lambda-parameters-with-modifiers.md [hunk 1]
```diff
@@ -75,6 +75,12 @@ to those that have no out parameters.
 + is restricted to those that have the same modifiers in the same order (§10.7).
 ```
 
+## Open LDM questions
+
+1. Currently (in C# 13 `(scoped x) => ...` means "A lambda with a parameter named 'x' with type 'scoped'".  We would like to change that to be treated as the "scoped modifier" instead.  We can gate this behind a language version change so it only breaks on upgrade.  There is also a suitable user workaround if they truly had this pathological code.  Namely, writing `(@scoped x) => ...`.
```
Comment (jaredpar) [inline proposals/simple-lambda-parameters-with-modifiers.md:80]: Agree make it a keyword on upgrade here.

**Review by Princeprince559 — COMMENTED** [review]
merged i asume... lol

**Pull request merged**